CRM/Logging/Schema.php

```diff
@@ -487,7 +487,7 @@ public function fixSchemaDifferencesForAll($rebuildTrigger = FALSE) {
   public function fixTimeStampAndNotNullSQL($query) {
     $query = str_ireplace("TIMESTAMP NOT NULL", "TIMESTAMP NULL", $query);
     $query = str_ireplace("DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP", '', $query);
```

Comment:
Should this `CURRENT_TIMESTAMP` be changed as well?

Reply:
According to the way I read the code, I think the second instance should take care of anything left behind from the first.  Perhaps it's safer, especially in the long run, to do both?